From: YOKOTA Hiroshi <yokota.hgml@gmail.com>
Date: Mon, 7 Jul 2025 11:53:15 +0900
Subject: Drop unused macro while building SFX stub

> ../../UI/Common/ArchiveExtractCallback.cpp:68:11: error: macro "REPLACE_SLASHES_from_Linux_to_Sys" is not used [-Werror=unused-macros]
>    68 |   #define REPLACE_SLASHES_from_Linux_to_Sys(s)
>       |           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
> cc1plus: all warnings being treated as errors
---
 CPP/7zip/UI/Common/ArchiveExtractCallback.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CPP/7zip/UI/Common/ArchiveExtractCallback.cpp b/CPP/7zip/UI/Common/ArchiveExtractCallback.cpp
index 88c3cf8..45ac1f5 100644
--- a/CPP/7zip/UI/Common/ArchiveExtractCallback.cpp
+++ b/CPP/7zip/UI/Common/ArchiveExtractCallback.cpp
@@ -58,6 +58,7 @@ static const char * const kCantCreateSymLink = "Cannot create symbolic link";
 
 static const unsigned k_LinkDataSize_LIMIT = 1 << 12;
 
+#ifndef Z7_SFX
 #if WCHAR_PATH_SEPARATOR != L'/'
   // we convert linux slashes to windows slashes for further processing.
   // also we convert linux backslashes to BackslashReplacement character.
@@ -67,6 +68,7 @@ static const unsigned k_LinkDataSize_LIMIT = 1 << 12;
 #else
   #define REPLACE_SLASHES_from_Linux_to_Sys(s)
 #endif
+#endif // Z7_SFX
 
 
 #ifndef Z7_SFX
